addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -426,6 +428,13 @@ QUnit.test('chat window: basic rendering', async function (assert) {
         "General",
         "should have correct thread name in header part"
     );
+
+    assert.doesNotHaveClass(
+        chatWindowHeader.querySelector('.o_ChatWindowHeader_name'),
+        'o-clickable',
+        "should not have clickable class on channel type chat window"
+    );
+
```

Comment:
Remove this. It doesn't have its place in a basic rendering test.
Prefer a dedicated test for that, if needed.

Also same remark as before: `o-clickable` is a (convenient) technical detail. Assertion comment should just be _"X should be clickable"_.

Reply:
This comment is still not resolved

Reply:
So as I see you added a new test, the change here can be reverted.